systemtests/tests/xattr/testrunner

```diff
@@ -0,0 +1,77 @@
+#!/bin/sh
+#
+# Run a simple backup
+#   then restore it.
+#
+TestName="$(basename "$(pwd)")"
+export TestName
+
+JobName=backup-bareos-fd
+
+#shellcheck source=../environment.in
+. ./environment
+
+#shellcheck source=../scripts/functions
+. "${rscripts}"/functions
+"${rscripts}"/cleanup
+"${rscripts}"/setup
+
+
+# Directory to backup.
+# This directory will be created by setup_data "$@"().
+BackupDirectory="${tmp}/data"
+BackupFile="${tmp}/file-with-xattr"
+RestoreFile="${tmp}/bareos-restores/${BackupFile}"
+
+xattr_should="${tmp}/xattr_should.txt"
+xattr_is="${tmp}/xattr_is.txt"
+
+mkdir -p "${BackupDirectory}"
+touch "${BackupFile}"
+setfattr --name=user.bareos.attr1 --value=value1 "${BackupFile}"
```

Comment:
```suggestion
setfattr --name=user.bareos.attr1 --value=value1 "${BackupFile}" || exit_error "Could not set attribute"
```

Reply:
Could add something like this exit_error function that makes it easier to see an error during script execution or add set -e.

Reply:
The code doesn't quit in that case intentionally.
Point being is that if you run the systemtests on a filesystem w/o xattr-support, the setfattr command will fail and you will end up with a file with no xattrs. 
While it doesn't check the xattrs really work, it at least doesn't fail.

The right thing is probably to disable the test in that case. However, I don't see an easy way to do that properly.

Reply:
"At least it doesn't fail" would lead to a false positive test. Couldn't you use a cmake command to verify if the setfattr command succeeds in general?

Reply:
imho it should be the same path as where the test will be running.

Reply:
That would be best, but that path is only created after we decided whether or not the test should be enabled.